ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.js

```diff
@@ -76,4 +79,174 @@ describe('Workload CVE overview page tests', () => {
             */
         });
     });
+
+    describe('Images without CVEs view tests', () => {
+        const cveListRouteMatcherMap = {
+            getImageCVEList: {
+                method: 'POST',
+                url: `/api/graphql?opname=getImageCVEList`,
+                times: 1,
+            },
+        };
+
+        const imageListRouteMatcherMap = {
+            getImageList: {
+                method: 'POST',
+                url: `/api/graphql?opname=getImageList`,
+                times: 1,
+            },
+        };
+
+        const deploymentListRouteMatcherMap = {
+            getDeploymentList: {
+                method: 'POST',
+                url: `/api/graphql?opname=getDeploymentList`,
+                times: 1,
+            },
+        };
+
+        beforeEach(function () {
+            if (!hasFeatureFlag('ROX_VULN_MGMT_NO_CVES_VIEW')) {
+                this.skip();
+            }
+        });
+
+        it('should remove cve-related UI elements when viewing the "without cves" view', () => {
+            visitWorkloadCveOverview();
+
+            // eslint-disable-next-line no-unused-vars
+            const riskPriorityHeader = 'th:contains("Risk priority")';
+            // eslint-disable-next-line no-unused-vars
+            const cvesBySeverityHeader = 'th:contains("CVEs by severity")';
```

Comment:
i feel like it might make more sense to just comment these consts out instead of having an eslint exception since the `cy.get` calls below are also commented out for now